src/lib/courses.ts

```diff
@@ -69,22 +69,83 @@ const courseQuery = groq`
 }
 `
 
+// TODO: go through the minimum Course page requirements, find the overlap with
+// what this Course document supports, and then lineup the naming.
+const courseQuery = groq`
+*[_type == 'course' && slug.current == $slug][0]{
```

Comment:
Still working on the shape of this groq query, some details here: https://roamresearch.com/#/app/egghead/page/R-ebGysU7